src/prefect/tasks/fivetran/__init__.py

```diff
@@ -0,0 +1,9 @@
+"""
+This module contains a task for starting and monitoring Fivetran connector sync jobs
+"""
+try:
```

Comment:
Unless I'm mistaken, your task doesn't require any special dependencies, so it will actually be available automatically to every Prefect user! You could remove this `try/except` if you prefer.

Reply:
I see! I was a little unclear on the purpose of this section of code. I'll pull that out.